c/nitf/include/nitf/System.h

```diff
@@ -62,15 +62,15 @@
 /******************************************************************************/
 /* DEBUG                                                                      */
 /******************************************************************************/
+#include "nrt/Debug.h"
 #ifdef NITF_DEBUG
     #define NRT_DEBUG 1
-    #include "nrt/Debug.h"
-
-    #define NITF_MEM_LOG        NRT_MEM_LOG
-    #define nitf_Debug_malloc   nrt_Debug_malloc
-    #define nitf_Debug_realloc  nrt_Debug_realloc
-    #define nitf_Debug_free     nrt_Debug_free
 #endif
+
```

Comment:
Why were these moved out of debug?

Reply:
They need to be defined, even if they don't do anything.  In modern C, you have to declare a function.  In the old days, you didn't have to.